cypress/e2e/main.cy.ts

```diff
@@ -7,5 +7,6 @@ describe("Homepage tests", () => {
     });
     cy.get("@consoleError").should("not.be.called");
     cy.get("body").should("exist");
+    cy.get("h1").should("exist");
```

Comment:
Why was this added?

Reply:
Not very relevant to this PR, just adding more coverage while I was testing, so I thought it could remain there.